### ISSUE TITLE
sbcl fails to build on Ubuntu 14.04

### ISSUE THREAD

**Issue by rwhogg** [Linuxbrew/linuxbrew#923]
`brew install sbcl` fails on Ubuntu 14.04. See https://gist.github.com/rwhogg/c750958273a5e0048a09

- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [x] If you're seeing permission errors tried running `sudo chown -R $(whoami) $(brew --prefix)`?

[Message 1](https://gist.github.com/rwhogg/c750958273a5e0048a09#file-01-make-sh-L7):
```
find: `../../obj/asdf-cache/sb-md5': No such file or directory
find: `../../obj/asdf-cache/sb-queue': No such file or directory
find: `../../obj/asdf-cache/sb-concurrency': No such file or directory
find: `../../obj/asdf-cache/sb-rotate-byte': No such file or directory
find: `../../obj/asdf-cache/sb-grovel': No such file or directory
find: `../../obj/asdf-cache/sb-sprof': No such file or directory
find: `../../obj/asdf-cache/sb-bsd-sockets': No such file or directory
find: `../../obj/asdf-cache/sb-cover': No such file or directory
find: `../../obj/asdf-cache/sb-posix': No such file or directory
```

[Message 2](https://gist.github.com/rwhogg/c750958273a5e0048a09#file-01-make-sh-L48)
```
�Y�: not found
/home/bob/tmp/sbcl--bootstrap6420160312-9981-1fk801v/sbcl-1.1.8-x86-64-darwin/src/runtime/sbcl: 1: /home/bob/tmp/sbcl--bootstrap6420160312-9981-1fk801v/sbcl-1.1.8-x86-64-darwin/src/runtime/sbcl: �����x�H__PAGEZEROx__TEXT: not found
/home/bob/tmp/sbcl--bootstrap6420160312-9981-1fk801v/sbcl-1.1.8-x86-64-darwin/src/runtime/sbcl: 2: /home/bob/tmp/sbcl--bootstrap6420160312-9981-1fk801v/sbcl-1.1.8-x86-64-darwin/src/runtime/sbcl: : not found
/home/bob/tmp/sbcl--bootstrap6420160312-9981-1fk801v/sbcl-1.1.8-x86-64-darwin/src/runtime/sbcl: 3: /home/bob/tmp/sbcl--bootstrap6420160312-9981-1fk801v/sbcl-1.1.8-x86-64-darwin/src/runtime/sbcl: Syntax error: "(" unexpected
Command exited with non-zero status 1
0.00user 0.00system 0:00.25elapsed 3%CPU (0avgtext+0avgdata 1656maxresident)k
0inputs+8outputs (0major+246minor)pagefaults 0swaps
```

**Comment by sjackman** [comment]
Can you please quote the relevant lines of the gist and link to them?

**Comment by rwhogg** [comment]
Comment updated.

**Comment by sjackman** [comment]
Triple back ticks are better for quoting code blocks than `>`.

**Comment by sjackman** [comment]
It downloads a Mac binary to bootstrap the system. It will have to instead download the Linux binary:
https://github.com/Linuxbrew/linuxbrew/blob/master/Library/Formula/sbcl.rb#L28

**Comment by rwhogg** [comment]
There's more to it than that - some of the patches are problematic too.

I've got it to build by disabling all the patches - now I've just got to work out the minimal subset to disable.

**Comment by rwhogg** [comment]
Fixed by #964
